modules/logging-loki-gui-install.adoc

```diff
@@ -75,10 +71,11 @@ include::snippets/logging-approval-strategy-snip.adoc[]
 The Operators should now be available to all users and projects that use this cluster.
 
 . Verify the operator installations:
-.. Navigate to *Operators* -> *Installed Operators*.
-.. Make sure the *openshift-logging* project is selected.
-.. In the *Status* column, verify that you see green checks with *InstallSucceeded* and the text *Up to date*, below.
-+
+. Navigate to *Operators* -> *Installed Operators*.
+. Make sure the *openshift-logging* project is selected.
+. In the *Status* column, verify that you see green checks with *InstallSucceeded* and the text *Up to date*, below.
```

Comment:
Please revise this sentence; we don't use "below" generally for accessibility per the IBMSG:
![image](https://github.com/openshift/openshift-docs/assets/40172997/eef8fb8d-a1b5-4c1a-a7f2-cb5ef5b14ffc)
```suggestion
. In the *Status* column, verify that you see green checkmarks with *InstallSucceeded* and the text *Up to date*.
```